google/cloud/automl/unit-tests/cloud_automl_vision_model_tests.yaml

```diff
@@ -0,0 +1,86 @@
+test:
+  suites:
+  - name: Cloud AutoML Vision Model API IT
+    setup:
+    - code: cloud_region = "us-central1"
+      # Read Project ID from Environment Variable
+    - env:
+        what: GOOGLE_PROJECT_ID
+        var: project_id
+    - env:
+        what: MODEL_ID
+        var: model_id
+    - env:
+        what: MODEL_DISPLAY_NAME
+        var: model_display_name
+    cases:
+    - name: List Model
+      spec:
+      - call: 
+          sample: automl_vision_list_models
+          params: 
+            location: { variable: cloud_region }
+            project_id: { variable: project_id }
+            filter: { literal: "" }
+      - assert_contains:
+        - literal: "Model name:"
+        - literal: "Training budget:"
+        - literal: "Training cost:"
+
+
+    - name: Get Model
+      spec:
+      - call: 
+          sample: automl_vision_get_model
+          params: 
+            location: { variable: cloud_region }
+            project_id: { variable: project_id }
+            model_id: { variable: model_id }
+      - assert_contains:
+        - literal: "Model display name:"
+        - variable: model_display_name
+
+    # TODO - listModelEvaluations
```

Comment:
What are these TODOs? Could you update with a more descriptive message? :)

Reply:
I just forgot to delete them afterwards